### ISSUE TITLE
Increase contrast option on mac breaks visibility of tabs in default white theme

### ISSUE THREAD

**Issue by k-kruglov** [brave/brave-browser#9816]
## Description 

Increase contrast option on mac breaks visibility of tabs in default white theme

## Steps to Reproduce

   1. Enable "increase contrast" option in mac settings - accessibility
   2. Open brave, enable default white theme
   

## Actual result:

![Screenshot 2020-05-17 at 21 58 53](https://user-images.githubusercontent.com/1358036/82157426-0715cf00-988a-11ea-85a2-e1fe6b281c94.png)
![Screenshot 2020-05-17 at 21 58 57](https://user-images.githubusercontent.com/1358036/82157427-0846fc00-988a-11ea-9013-c55bb3c7f428.png)

If window is active, tabs text will be barely visible

## Expected result:

Tabs text is clearly visible

## Reproduces how often: 

Every the time, easily reproduced

## Version/Channel Information:

- Can you reproduce this issue with the current release? - yes, Version 1.8.96
- Can you reproduce this issue with the beta channel? - haven't tried
- Can you reproduce this issue with the dev channel? - haven't tried
- Can you reproduce this issue with the nightly channel? - haven't tried

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? no
- Does the issue resolve itself when disabling Brave Rewards? no
- Is the issue reproducible on the latest version of Chrome? haven't tried

## Miscellaneous Information:

Brave Version 1.9.72
mac os catalina 10.15.4

**Comment by bxpana** [comment]
I've found the same issue and had to quit Brave after disabling the increase of contrast. This seems like an error seeing that people that want that contrast at all times will not be able to read the tabs bar.

**Comment by k-kruglov** [comment]
As a workaround I'm currently using black and white theme (which is a very contrasty one).

https://chrome.google.com/webstore/detail/black-white/mhhlgkfginnlendpfkhcmldikeepoefa

**Comment by k-kruglov** [comment]
Fixed in 1.10.90
